.scalafix.conf

```diff
@@ -46,5 +46,5 @@ SortImports.blocks = [
   "scala.",
   "*",
   "com.sun."
-  "akka."
+  "org.apache.conf."
```

Comment:
should this be "org.apache.pekko."? Could you keep "akka." too - because pekko-http and other repos like that may keep their akka dependencies for a small period of time - until we have regular nightly jars of the core pekko modules.

Reply:
> should this be "org.apache.pekko."? 

Thanks for pickup, done

> Could you keep "akka." too - because pekko-http and other repos like that may keep their akka dependencies for a small period of time - until we have regular nightly jars of the core pekko modules.

This has nothing to do with dependencies, its used by scalafix to sort imports and none of the imports in our code will ever have `akka` in it (thats actually what this PR changes).